src/components/molecules/Visualizer/Widget/Navigator/index.tsx

```diff
@@ -0,0 +1,16 @@
+import NavigatorUI from "@reearth/components/atoms/Navigator";
+import { ComponentProps as WidgetProps } from "@reearth/components/molecules/Visualizer/Widget";
+
+import { SceneProperty } from "../../Engine";
+
+import { useNavigator } from "./hooks";
```

Comment:
We usually just name hooks `useHooks`. So that we don't have to look at the import to know they are the component's local hooks (and not, for example, third-party hooks) can you rename this?

Reply:
Using default exports for hooks specific to that component will also have the effect of preventing it from unnecessarily appearing as a candidate in code editor completion.

Reply:
This was fixed in [refactor: rename to useHooks](https://github.com/reearth/reearth-web/pull/323/commits/2289ecb95ee2438ef1caf8f53c2b3230933dd1a0)